raiden-cli/src/index.ts

```diff
@@ -311,6 +329,7 @@ function createRaidenConfig(
     pfsMaxPaths: argv.pathfindingMaxPaths,
     pfsMaxFee: argv.pathfindingMaxFee,
     pfsIouTimeout: argv.pathfindingIouTimeout,
+    ...(argv.gasPrice ? { gasPriceFactor: argv.gasPrice } : undefined),
```

Comment:
Shouldn't we set a factopr of 1.0 instead of `undefined` if nothing is given?

Reply:
I wouldn't say so, let's leave the unset case to the SDK, and there the default is 1.0, so we don't have to keep track of the default in multiple places

Reply:
Sounds good, I missed the spread operator there.